### DISCUSSION TITLE
How to load picture from device srorage?

### DISCUSSION THREAD

**Discussion by DmitryBorodiy** [AvaloniaUI/Avalonia#15732] [Q&A] [answered]
How I can load picture Bitmap from the disk and show it in Image using Avalonia?

Should:
  - Pick file with open file dialog
  - Load Bitmap from file
  - Display Bitmap in the Image control

**Comment by thevortexcloud** [comment]
Make an `Image` control and bind it to a `Bitmap`. Use the StorageProvider to get the file path.

https://docs.avaloniaui.net/docs/concepts/services/storage-provider/

**Reply by DmitryBorodiy** [reply]
So, Can Bitmap load from stream or full file path?

**Reply by thevortexcloud** [reply, marked as answer]
Yes. There is an example here: https://docs.avaloniaui.net/docs/tutorials/music-store-app/displaying-images

Otherwise just use the constructors:
https://reference.avaloniaui.net/api/Avalonia.Media.Imaging/Bitmap/CF219A66
https://reference.avaloniaui.net/api/Avalonia.Media.Imaging/Bitmap/EFE51BA5
